### PR TITLE
[CC][4.1.0] Adding docs for Choreo Connect updating

### PR DESCRIPTION
**Pull request by VajiraPrabuddhaka**
## Purpose
This PR introduces a new section for choreo-connect docs regarding the Choreo-Connect updating.

## Screenshots
screencapture-localhost-8000-deploy-and-publish-deploy-on-gateway-choreo-connect-update-choreo-connect-2022-07-27-17_54_17

### CODE REVIEW

File: en/mkdocs.yml [hunk 1]
```diff
@@ -358,6 +358,7 @@ nav:
                     - Error Handling: deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/error-handling.md
                     - Enforcer Admin API: deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/enforcer-admin-api/enforcer-v1.0.0.md
                     - Debug Memory Issues: deploy-and-publish/deploy-on-gateway/choreo-connect/troubleshooting/debug-memory.md
+                - Updating Choreo Connect: deploy-and-publish/deploy-on-gateway/choreo-connect/update-choreo-connect.md
```
Comment (pubudu538) [inline en/mkdocs.yml:361]: SHould be Update Choreo Connect

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/update-choreo-connect.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Update Choreo Connect
+
+WSO2 introduces [WSO2 Updates](https://updates.docs.wso2.com/en/latest/) , which is a command-line utility that allows you to get the latest updates that are available for a particular product release. These updates include the latest bug fixes and security fixes that are released by WSO2 after a particular product version is released. Therefore, you do not need to wait and upgrade to the next product release to get these bug fixes.
+
```
Comment (pubudu538) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/update-choreo-connect.md:4]: Above Para is not required.

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/update-choreo-connect.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+You can easily upgrade Choreo Connect [Using WSO2 Updates 2.0](https://updates.docs.wso2.com/en/latest/updates/update-tool/). The docker images related to Choreo Connect updates are reside in [WSO2 private docker registry](https://docker.wso2.com/) and the image tags are having the following format for Choreo Connect components.
```
Comment (pubudu538) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/update-choreo-connect.md:5]: We should include the following.

1st Para
1. Public images - https://hub.docker.com/u/wso2
2. If you have a WSO2 subscription you can use the latest docker images with updates. These updates include the latest bug fixes and security fixes that are released by WSO2 after a particular product version is released. https://docker.wso2.com/

Topic: Docker Image Tags
You can have the below table.

Topic: Update Portal

You can use https://updates-info.wso2.com/ with sub details to login into update portal. You will be able to see the information like General description, Implementation details, Impact, Bug fixes and specific Instructions. Read more info - https://updates.docs.wso2.com/en/latest/updates/updates-portal/

File: en/mkdocs.yml (hunk 1 above)
Comment (VajiraPrabuddhaka) [reply en/mkdocs.yml]: fixed via https://github.com/wso2/docs-apim/pull/6096/commits/ff7abbd3615e65f8560ce14ffc15025f6c9325fc

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/update-choreo-connect.md (hunk 3 above)
Comment (VajiraPrabuddhaka) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/update-choreo-connect.md]: fixed via https://github.com/wso2/docs-apim/pull/6096/commits/ff7abbd3615e65f8560ce14ffc15025f6c9325fc

**Review by pubudu538 — APPROVED** [review]

**Comment by pubudu538** [comment]
Let's get the same for 4.0.0 and main branch

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/update-choreo-connect.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
+# Update Choreo Connect
+
+Choreo Connect components are available as public docker images in [WSO2](https://hub.docker.com/u/wso2) organization in the docker hub whereas you will be able to find images corresponding to [Choreo Connect releases](https://github.com/wso2/product-microgateway/releases).
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/update-choreo-connect.md:3]:
```suggestion
Choreo Connect components are available as public Docker images in [WSO2](https://hub.docker.com/u/wso2) organization in the Docker hub whereas you will be able to find images corresponding to [Choreo Connect releases](https://github.com/wso2/product-microgateway/releases). 
```

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/update-choreo-connect.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
+# Update Choreo Connect
+
+Choreo Connect components are available as public Docker images in [WSO2](https://hub.docker.com/u/wso2) organization in the Docker hub whereas you will be able to find images corresponding to [Choreo Connect releases](https://github.com/wso2/product-microgateway/releases). 
+
+However, if you have a WSO2 Subscription you will be able to use latest docker images with updates, which includes bug fixes, features, improvements, etc. on top of genarally available releases. This documentation is about how you can use that particular updates.
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/update-choreo-connect.md:5]:
```suggestion
However, if you have a WSO2 Subscription you will be able to use latest Docker images with updates, which includes bug fixes, features, improvements, etc. on top of generally available releases. This documentation is about how you can use those particular updates.
```

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/update-choreo-connect.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
+# Update Choreo Connect
+
+Choreo Connect components are available as public Docker images in [WSO2](https://hub.docker.com/u/wso2) organization in the Docker hub whereas you will be able to find images corresponding to [Choreo Connect releases](https://github.com/wso2/product-microgateway/releases). 
+
+However, if you have a WSO2 Subscription you will be able to use latest Docker images with updates, which includes bug fixes, features, improvements, etc. on top of generally available releases. This documentation is about how you can use those particular updates.
+
+## Docker Image Tags
+
+The docker images related to Choreo Connect updates are reside in [WSO2 private docker registry](https://docker.wso2.com/) and the image tags are having the following format for Choreo Connect components.
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/update-choreo-connect.md:9]:
```suggestion
The Docker images related to Choreo Connect updates reside in [WSO2 private Docker registry](https://docker.wso2.com/) and the image tags have the following format for Choreo Connect components.
```

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/update-choreo-connect.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+                <code>choreo-connect-adapter:1.1.0.3</code>
+            </td>
+        </tr>  
+    </tbody>
+</table>
+
+!!! Info
+    For more details on versioning, you may follow through the documentation on [Docker image tags with Updates 2.0](https://updates.docs.wso2.com/en/latest/updates/using-wso2-docker-images/).
+
+You will be able to use **Updates Portal** to see more details about these update levels in docker image tags, which will be covered in next section.
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/update-choreo-connect.md:59]:
```suggestion
You will be able to use the **Updates Portal** to see more details about these update levels in Docker image tags, which will be covered in the next section.
```

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/update-choreo-connect.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+</table>
+
+!!! Info
+    For more details on versioning, you may follow through the documentation on [Docker image tags with Updates 2.0](https://updates.docs.wso2.com/en/latest/updates/using-wso2-docker-images/).
+
+You will be able to use the **Updates Portal** to see more details about these update levels in Docker image tags, which will be covered in the next section.
+
+## Updates Portal
+
+Use your subscription details to login to the Updates Portal via [https://updates-info.wso2.com](https://updates-info.wso2.com/). You will be able to see the information like General description, Implementation details, Impact, Bug fixes and specific Instructions on each update in the **Update Levels** section by choosing Choreo Connect as the product and appropriate product version.
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/update-choreo-connect.md:63]:
```suggestion
Use your subscription details to log in to the Updates Portal via [https://updates-info.wso2.com](https://updates-info.wso2.com/). You will be able to see information like General description, Implementation details, Impact, Bug fixes, and specific Instructions on each update in the **Update Levels** section by choosing Choreo Connect as the product and the appropriate product version.
```

**Review by samgnaniah — APPROVED** [review]

**Pull request merged**